nb

```diff
@@ -440,8 +440,15 @@ _option_value_is_present() {
 #
 # Default: `$HOME/.nbrc`
 #
-# The location of the .nbrc configuration file.
-export NBRC_PATH="${NBRC_PATH:-"${HOME}/.${_ME}rc"}"
+# Check if XDG_CONFIG_HOME exists
+if [[ -n "${XDG_CONFIG_HOME}" ]]; then
+  # If XDG_CONFIG_HOME exists put .nbrc config file in a subdirectory.
+  mkdir -p "${XDG_CONFIG_HOME}/${_ME%%.*}" &&
+  export NBRC_PATH="${NBRC_PATH:-"${XDG_CONFIG_HOME}/${_ME%%.*}/.${_ME}rc"}"
```

Comment:
I have commonly seen the main config file for programs installed under `$XDG_CONFIG_DIR/subdir` named `config` instead of something specific to the program like `.nbrc`. It may be worth considering here.
i.e.
```suggestion
  export NBRC_PATH="${NBRC_PATH:-"${XDG_CONFIG_HOME}/${_ME%%.*}/config"}"
```